### PR TITLE
DOCSP-33948: Atomic typos and cleanup

### PR DESCRIPTION
**Pull request by norareidy**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-node/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-33948>
Staging - <https://preview-mongodbnorareidy.gatsbyjs.io/node/DOCSP-33948-atomic-typos/>

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/fundamentals/crud/compound-operations.txt [hunk 1]
```diff
@@ -13,9 +20,9 @@ Compound Operations
 Overview
 --------
 
-Most database requests need to either read data out of a database or
-write data into a database. However, there are instances where you may
-need to read and write data in a single interaction.
+Most database requests either read data out of a database or write data into
```
Comment (mongoKart) [inline source/fundamentals/crud/compound-operations.txt:23]: S:

```suggestion
Most database requests either read data from a database or write data into
```

File: source/fundamentals/crud/read-operations/project.txt [hunk 2]
```diff
@@ -71,11 +71,10 @@ field of each document:
...
 The projection document specifies a value of ``1`` for ``name`` to
-indicate that the read operation result should *include* the ``name``
-field of each returned document. As a result, this projection implicitly
-excludes the ``qty`` and ``rating`` fields. Passing this projection to
-``find()`` with an empty query document and no sort document yields
-the following results:
+instruct the operation to *include* the ``name`` field of each returned document in
+the results. As a result, this projection implicitly excludes the ``qty`` and
+``rating`` fields. Passing this projection to ``find()`` with an empty query document
+and no sort document yields the following results:
```
Comment (mongoKart) [inline source/fundamentals/crud/read-operations/project.txt:77]: S: sounds a little funny to say "result" twice so close together with different meanings

you could end the first sentence after ``name``, then pick up here:

```suggestion
This instructs the operation to *include* the ``name`` field of each returned document in
the results and *exclude* the ``qty`` and
``rating`` fields. Passing this projection to ``find()`` with an empty query document
and no sort document yields the following results:
```

File: source/fundamentals/crud/read-operations/project.txt [hunk 3]
```diff
@@ -111,8 +110,8 @@ returned documents.
    }
 
 The projection document specifies a value of ``1`` for ``name`` to
-indicate that the read operation result should *include* the ``name``
-field of each returned document. As a result, this projection implicitly
+instruct the operation to *include* the ``name`` field of each
+returned document in the results. As a result, this projection implicitly
 excludes the ``qty`` and ``rating`` fields. Passing this projection to
```
Comment (mongoKart) [inline source/fundamentals/crud/read-operations/project.txt:115]: Same suggestion here as above

File: source/fundamentals/crud/write-operations/upsert.txt [hunk 4]
```diff
@@ -16,8 +21,8 @@ Overview
 --------
 
 If your application stores and modifies data in MongoDB, you probably use
-insert and update operations. In certain workflows, you may need to choose
-between an insert and update depending on whether the document exists.
+insert and update operations. In certain workflows, you may want to perform
```
Comment (mongoKart) [inline source/fundamentals/crud/write-operations/upsert.txt:24]: I: [might/may](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#std-term-may:~:text=than%20matrices.-,may,-Use%20may%20to)

```suggestion
insert and update operations. In certain workflows, you might want to perform
```

File: source/fundamentals/crud/write-operations/upsert.txt [hunk 5]
```diff
@@ -16,8 +21,8 @@ Overview
...
+an insert or update operation depending on whether the document exists.
```
Comment (mongoKart) [inline source/fundamentals/crud/write-operations/upsert.txt:25]: S: clarity

```suggestion
insert and update operations. In certain workflows, whether you perform
an insert or update operation depends on whether the document exists.
```

File: source/fundamentals/stable-api.txt [hunk 6]
```diff
@@ -16,7 +21,7 @@
 
    The {+stable-api+} feature requires MongoDB Server 5.0 or later.
 
-   You should only use the {+stable-api+} feature if all the MongoDB
+   Only use the {+stable-api+} feature if all the MongoDB
```
Comment (mongoKart) [inline source/fundamentals/stable-api.txt:24]: I: [only](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#std-term-may:~:text=Don%27t%20hyphenate.-,only,-Place%20only%20preceding)

```suggestion
   Use the {+stable-api+} feature only if all the MongoDB
```

File: source/fundamentals/stable-api.txt [hunk 7]
```diff
@@ -51,10 +56,10 @@ version of the {+stable-api+}.
 
 .. tip::
 
-   If you need to run commands using more than one version of the
-   {+stable-api+}, instantiate a separate client with that version.
+   To run commands using more than one version of the {+stable-api+},
+   instantiate a separate client with that version.
```
Comment (mongoKart) [inline source/fundamentals/stable-api.txt:60]: S: clarity

```suggestion
   You must create a new client for each version of the {+stable-api+} on
   which you need to run a command.
```

(I think this is accurate? ^^^^)

File: source/includes/localhost-connection.rst [hunk 8]
```diff
@@ -1,5 +1,5 @@
-If you need to run a MongoDB server on your local machine for development
-purposes instead of using an Atlas cluster, you need to complete the following:
+If you want to run a MongoDB server on your local machine instead of
+using an Atlas cluster, complete the following steps:
```
Comment (mongoKart) [inline source/includes/localhost-connection.rst:2]: S: the intro text for this section (on the actual docs page) makes it clear that this is an alternative to Atlas

there was also a ticket for Java(?) some time ago that encouraged "MongoDB Server" as a product name and "MongoDB deployment" for individual instances, iirc

```suggestion
To connect to a MongoDB deployment on your local machine,
complete the following steps:
```

**Review by mongoKart — APPROVED** [review]
really nice job shortening some of the sentences. a few changes, but i don't need to see it again unless you have questions

File: source/fundamentals/stable-api.txt (hunk 7 above)
Comment (norareidy) [reply source/fundamentals/stable-api.txt:60]: seems right to me, changed!

**Pull request merged**